### DISCUSSION TITLE
本酋长于今早用大保底强娶了飞霄

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#4786] [崩坏：星穹铁道]
![Screenshot_2024-09-10-10-18-26-882_com hykb yuanshenmap](https://github.com/user-attachments/assets/769b934a-fcfd-4541-a5c6-89dadaf5e598)望周知T^T

**Comment by lllliufeng** [comment]
已阅

**Comment by The-Place-Where-Winter-Sieeps** [comment]
没事我两个都吃满了。飞霄170武器160<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by yangyae** [comment]
阅
